### DISCUSSION TITLE
Extending jackson object mapper

### DISCUSSION THREAD

**Discussion by almyy** [Netflix/dgs-framework#77] [Q&A] [answered]
I have a mutation with an argument of type `LocalDate` together with a scalar and corresponding Coercing. When calling the mutation, I get an error: `Cannot construct instance of 'java.time.LocalDate' (no Creators, like default constructor, exist): cannot deserialize from Object value` 

Seems like dgs is trying to run `jacksonObjectMapper().convertValue(parameterValue, parameter.type)` where both `parameterValue` and `parameter.type` is of the type LocalDate. I assume this would work if I could somehow register `JavaTimeModule` to the object mapper, but it seems like I can't control this in any way.

Any ideas? It would be nice to be able to extend the object mapper somehow, or maybe dgs shouldn't try to convert a value when the `fromType` is the same as the `toType`?

**Comment by srinivasankavitha** [comment, marked as answer]
This PR should address your issue: https://github.com/Netflix/dgs-framework/pull/72. The changes should be available soon.

**Reply by almyy** [reply]
Cool, thanks!
